src/RouteBinderServiceProvider.php

```diff
@@ -32,9 +45,10 @@ public function bootRoutes()
         /** @type \Illuminate\Routing\Router $router */
         $router = $this->app['router'];
 
-        foreach ($config->get('route-binder::routes') as $binder)
-        {
-            $this->app->make($binder)->bind($router);
+        if ($config->has('route-binder')) {
```

Comment:
@DraperStudio is this with L5?
I don't seem to have this issue with L4, I'll see if I can test with L5 tomorrow!